SwiftVocabulary/SwiftVocabulary/Base.lproj/Main.storyboard

```diff
@@ -26,7 +26,7 @@
                                         <label opaque="NO" multipleTouchEnabled="YES" contentMode="left" insetsLayoutMarginsFromSafeArea="NO" text="Title" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" id="9Jc-Ja-R8Y">
                                             <rect key="frame" x="20" y="0.0" width="374" height="80"/>
                                             <autoresizingMask key="autoresizingMask"/>
-                                            <fontDescription key="fontDescription" type="system" pointSize="17"/>
+                                            <fontDescription key="fontDescription" type="system" pointSize="25"/>
```

Comment:
I like the bigger font, looks nice. I can tell you will do well with user interface, your background comes into play there.